.tours/codeql-tutorial.tour

```diff
@@ -12,7 +12,8 @@
     },
     {
       "directory": "codeql-tutorial-database",
-      "description": "To run a CodeQL query, we need to select a CodeQL database.\n\nWe have prepared a sample database for you to use in this tutorial.\nLook for the `codeql-tutorial-database` directory highlighted in the Explorer view in your left sidebar.\n\nTo select this as your current database, right-click on the directory, and click `CodeQL: Set Current Database`. Click `Next` when you have done that."
+      "description": "To run a CodeQL query, we need to select a CodeQL database.\n\nWe have prepared a sample database for you to use in this tutorial, namely the `codeql-tutorial-database` directory.\n\nWe have selected this as your current database.",
```

Comment:
Minor: it might be useful to remind users 1) how to see which DB is selected and 2) how to select a different one.